modules/ROOT/nav.adoc

```diff
@@ -0,0 +1,35 @@
+* xref:getting-started.adoc[Getting Started]
+* xref:base-images.adoc[Base images]
+* Building containers
+** xref:building-containers.adoc[Building derived container images]
+** xref:building-custom-base.adoc[Building custom base images]
+* Provisioning machines
+** xref:bare-metal.adoc[Installing on Bare Metal]
+** xref:podman-bootc-cli.adoc[Local virt testing with podman-bootc-cli]
+** xref:qemu-and-libvirt.adoc[Using generic libvirt and qemu]
+** xref:provisioning-aws.adoc[Installing on AWS]
+** xref:provisioning-vsphere.adoc[Installing on vSphere]
+** xref:provisioning-gcp.adoc[Installing on GCP (via OpenTofu)]
+** xref:linux-desktops.adoc[Linux desktops]
```

Comment:
not sure of the best place for this page but under under provisioning machines seems like not the right place